docs/ref/contrib/admin/index.txt

```diff
@@ -1097,20 +1097,21 @@ subclass::
     ``['first_name', 'last_name']`` and a user searches for ``john lennon``,
     Django will do the equivalent of this SQL ``WHERE`` clause::
 
-        WHERE (first_name ILIKE '%john%' OR last_name ILIKE '%john%')
-        AND (first_name ILIKE '%lennon%' OR last_name ILIKE '%lennon%')
+        WHERE (first_name LIKE '%john%' OR last_name LIKE '%john%')
+        AND (first_name LIKE '%lennon%' OR last_name LIKE '%lennon%')
 
     For faster and/or more restrictive searches, prefix the field name
     with an operator:
 
     ``^``
-        Matches the beginning of the field. For example, if ``search_fields``
-        is set to ``['^first_name', '^last_name']`` and a user searches for
+        Use the '^' operator to match starting at the beginning of the
```

Comment:
'Matches the beginning of the field.' is a sentence fragment, and it stems from the layout of having a operator symbols, here the caret (^) and further down the = and @ on their own lines, The presentation is confusing. The point of the edited sentences following each of the otherwise extraneous operator symbols is explain what 'operator' is being referenced.